main.js

```diff
@@ -1221,10 +1259,15 @@ ipc.on('close-about', () => {
   }
 });
 
+ipc.on('close-stay-in-background-window', () => {
+  if (stayInBackgroundWindow) {
+    stayInBackgroundWindow.close();
+  }
+});
+
+
 ipc.on('update-tray-icon', (event, unreadCount) => {
-  if (tray) {
```

Comment:
As with earlier comment, this check that tray is truthy should stay because the tray is initialized after the window, so this could be called whilst tray is still null, causing a crash.